virttest/qemu_devices/qcontainer.py

```diff
@@ -2256,15 +2257,16 @@ def serials_define_by_variables(self, serial_id, serial_type, chardev_id,
                             for _ in bus_extra_params.split(',') if _]))
         # For virtio devices, generate controller and create the port device
         if serial_type.startswith('virt'):
-            if not bus:
+            if not bus or bus == 'new':
```

Comment:
To distinguish with other literal string values, suggest using `<new>`

Reply:
How about `new_bus`?

Reply:
> To distinguish with other literal string values, suggest using `<new>`

Thanks and update, please help check.

Reply:
@luckyh  Could you help review again, thanks!